cmd/dagger/engine.go

```diff
@@ -53,10 +53,6 @@ func withEngineAndTUI(
 	}
 
 	if silent {
-		if engineConf.LogOutput == nil {
```

Comment:
@vito I noticed this as I was just backfilling tests for project dirs being imported; I am setting the `silent` flag when invoking dagger do in the integ tests but I was still getting progress output.

Was setting LogOutput even when this is set intentional? I may have just misunderstood the docs for it of `disable terminal UI and progress output`

Reply:
Eh actually this broke something else, so I reverted and am leaving in those lines for now. Can address separately if updates are needed.

Reply:
Haha, I saw this a bit ago and realized it's probably a mistake. It's because I conflated "disable TUI" with `--silent` which feels totally wrong in hindsight. `--silent` should really mean silent, and we probably need some other flag to force console output. I'm working in this area already so I can fix it up along the way, but feel free to make any changes needed to unblock this PR.